lib/main_containers.pm

```diff
@@ -82,15 +82,15 @@ sub load_host_tests_docker {
         loadtest 'containers/registry' if is_x86_64;
         loadtest 'containers/docker_compose';
     }
-    loadtest 'containers/validate_btrfs' if (is_x86_64);    # works currently only for x86_64, more are coming (poo#103977)
+    loadtest 'containers/validate_btrfs' if (is_x86_64 and !is_jeos);    # works currently only for x86_64, more are coming (poo#103977)
```

Comment:
Why?!? I just fixed this in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13860 ([poo#102377](https://progress.opensuse.org/issues/102377))

Reply:
first reason is that the module is not scheduled on the removed yaml.
second the attempt to run VR with it, failed in all them. Link in the description still points to those jobs

Reply:
As the matter of fact, there are also modules which were not running before other than _containers/validate_btrfs_. i left them untouched but i am expecting some inputs if they are needed or not

Reply:
Well, since the VRs are only 15-SP4 and they are faiiling because `NUMDISK=1` there, how about we exclude it also only 15-SP4 JeOS?

```suggestion
    loadtest 'containers/validate_btrfs' if (!is_x86_64 and !(is_jeos && is_sle("=15-SP4")));    # works currently only for x86_64, more are coming (poo#103977)
```

Reply:
*this change is ready for review*

Reply:
However i just tried to run a verification and i remembered that the particular scheduling doesnt exist in the other products.